src/assets/images/light/AppFooter.vue

```diff
@@ -0,0 +1,141 @@
+<script setup lang="ts">
+import { computed } from 'vue'
+import { useRoute } from 'vue-router'
+
+import { useAppStore } from '@/store/app'
+import { useRequestStore } from '@/store/request'
+import { useUserStore } from '@/store/user'
+import { ChainType } from '@/utils/chainType'
+import { getImage } from '@/utils/getImage'
+
+const userStore = useUserStore()
+const route = useRoute()
+const requestStore = useRequestStore()
+const appStore = useAppStore()
+
+const currentRoute = computed(() => {
+  return route.name
+})
+
+function isCurrentRoute(pathName) {
+  return currentRoute.value === pathName
+}
+
+function getIcon(icon: string, pathName: string) {
+  if (isCurrentRoute(pathName)) {
+    return getImage(`${icon}-selected.svg`)
+  } else {
+    return getImage(`${icon}.svg`)
+  }
+}
+</script>
+
+<template>
+  <footer
+    class="flex items-center px-12 py-2 bg-gray-900 border-gray-800 border-t-1 dark:bg-black-400 dark:border-0 select-none"
+  >
+    <nav v-if="userStore.isLoggedIn" class="flex flex-grow justify-between">
+      <router-link
+        :to="{ name: 'home' }"
+        title="Assets"
+        class="flex flex-col justify-center items-center gap-1"
+      >
+        <div
+          class="w-xxxl h-xxl rounded-md flex items-center justify-center"
+          :class="{ 'bg-[#1D2A31] dark:bg-[#F7F7F7]': isCurrentRoute('home') }"
+        >
+          <img
+            :src="getIcon('tokens-icon', 'home')"
+            alt="home"
+            class="w-xl h-xl"
+          />
+        </div>
+        <span
+          class="text-xs font-norma"
```

Comment:
what is font-norma?

Reply:
[fixed](https://github.com/arcana-network/wallet-ui/pull/693/commits/f997475ff9d23b36ea41e8217957c2029e930b46)